documentation/release_notes.rst

```diff
@@ -8,6 +8,34 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.7.0
+===============
+
+Fixed Issues
+------------
+- Fixed a kernel name mangling error. The error arose with range-based algorithms or ``reduce_by_segment`` used with
+  `dpcpp_default` or another DPC++ policy object constructed with no explicitly provided kernel name.
+  
+Known Issues and Limitations
+----------------------------
+New in This Release
+^^^^^^^^^^^^^^^^^^^
+- An application not using Parallel STL algorithms in GNU C++ standard library (libstdc++) version 9 may not compile with C++17/20
+  if oneTBB is not present in the environment. Disable support for Parallel STL algorithms by setting
+  the macro ``PSTL_USE_PARALLEL_POLICIES`` to zero before including the first standard header file in each translation unit.
+  Or use the workaround in the oneDPL library by including the oneDPL headers before the rest of the headers.
```

Comment:
This known issue is about something else. But his wording is misleading because it looks like a note in the library guide. I think this would describe the problem better:
```suggestion
- When using oneDPL with serial/openMP backend and libstdc++ version 9, C++17/20, a compilation error occurs
if oneTBB is not present in the environment. It happens because libstdc++ version 9 does not check for
oneTBB availability. To get rid of the error, disable support for Parallel STL algorithms by setting the macro
PSTL_USE_PARALLEL_POLICIES to zero before including the first standard header file in each translation unit.
Or use the workaround in the oneDPL library by including the oneDPL headers before the rest of the headers.
```

Reply:
Anyway, it's not a temporary issue but a known limitation which we cannot fix - so the description of it should be in the library guide, either combined with or next to the other libstdc++ & TBB issue.

Reply:
Moved this to the library guide https://github.com/oneapi-src/oneDPL/pull/545